tests/adapters/vip-search/VipSearchPhraseMatchingTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Elasticsearch Extensions Tests: VIP_Enterprise_Search_Adapter_TestAPI class.
+ *
+ * @package Elasticsearch_Extensions
+ * @subpackage Tests
+ */
+use Elasticsearch_Extensions\Factory;
+
+it( 'tests that phrase matching matches phrase exactly.', function () {
+	// Adapter is loaded in the VIP Unit Test Case.
+	// Enable phrase matching.
+	elasticsearch_extensions()->load_adapter( Factory::vip_enterprise_search_adapter() );
+	elasticsearch_extensions()->enable_phrase_matching();
+
+	// Run a search for the phrase "exact match" and get the global wp_query.
+	$this->get( '/?s="exact match"' );
+	global $wp_query;
+
+	// Only 1 of the 3 posts should be found with hits for the words "exact match".
+	$this->assertEquals( 1, $wp_query->found_posts );
```

Comment:
I'd take this a step further and ensure the post that matched is the one that we wanted.